### PR TITLE
Pull request for feature/method for adding new patient into the system

### PR DESCRIPTION
**Pull request by AndriiKatsubo**

### CODE REVIEW

**Review by Ravennod — APPROVED** [review]

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    [Table("ChildrenCards")]
+    public class ChildCard
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column("id")]
+        public int Id { get; set; }
+
+        [StringLength(100)]
+        [Column("f_name")]
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Why did you write this `Column` attribute?

Comment (AndriiKatsubo) [reply MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: I wrote this `Column `attribute to specify name of column in database.

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        public string F_Name { get; set; }
+
+        [StringLength(100)]
+        [Column("s_name")]
+        public string S_Name { get; set; }
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Code convention!!!!!!!!!!!!!!!!!!
For properties you must use pascal case

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MedicalDocManagment.MainAppDAL.Entities
+{
+    [Table("ChildrenCards")]
+    public class ChildCard
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column("id")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MedicalDocManagment.MainAppDAL.Entities
+{
+    [Table("ChildrenCards")]
+    public class ChildCard
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: It's redundant attribute

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+        [StringLength(100)]
+        [Column("s_name")]
+        public string S_Name { get; set; }
+
+        [StringLength(100)]
+        [Column("l_name")]
+        public string L_Name { get; set; }
+
+        [Column("date")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        public DateTime Date { get; set; }
+
+        [Column("address")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        public string Address { get; set; }
+
+        [Column("checkin")]
+        public DateTime CheckIn { get; set; }
+
+        [Column("checkout")]
+        public DateTime? CheckOut { get; set; }
+
+        [Column("id_diagnose")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        [Column("date")]
+        public DateTime Date { get; set; }
+
+        [Column("address")]
+        public string Address { get; set; }
+
+        [Column("checkin")]
+        public DateTime CheckIn { get; set; }
+
+        [Column("checkout")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+        [Column("l_name")]
+        public string L_Name { get; set; }
+
+        [Column("date")]
+        public DateTime Date { get; set; }
+
+        [Column("address")]
+        public string Address { get; set; }
+
+        [Column("checkin")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        [Column("id_diagnose")]
+        public string DiagnosisId { get; set; }
+
+        [ForeignKey("DiagnosisId")]
+        public virtual Diagnosis Diagnosis { get; set; }
+
+        [Column("prescription")]
+        public string Prescription { get; set; }
+
+        [Column("directedby")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        [Column("checkout")]
+        public DateTime? CheckOut { get; set; }
+
+        [Column("id_diagnose")]
+        public string DiagnosisId { get; set; }
+
+        [ForeignKey("DiagnosisId")]
+        public virtual Diagnosis Diagnosis { get; set; }
+
+        [Column("prescription")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/ChildCard.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/Diagnosis.cs [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNet.Identity.EntityFramework;
+
+namespace MedicalDocManagment.MainAppDAL.Entities
+{
+    [Table("Diagnoses")]
+    public class Diagnosis
+    {
+        [Key]
+        [Column("id")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/Diagnosis.cs]: Column atribute is redundant

File: MedicalDocManagment.MainAppDAL/Entities/Parent.cs [hunk 13]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MedicalDocManagment.MainAppDAL.Entities
+{
+    [Table("Parents")]
```
Comment (artempanko) [inline MedicalDocManagment.MainAppDAL/Entities/Parent.cs]: Table atribute is redundant

**Review by KostetskyiYaroslav — APPROVED** [review]

**Pull request merged**